test/evergreen.yml

```diff
@@ -5822,8 +5822,6 @@ buildvariants:
       distros: ubuntu2004-arm64-large
     - name: format-smoke-test
     - name: format-stress-pull-request-test
-    - name: linux-directio
```

Comment:
What's the context for this? It seems a little odd to remove these tests without removing the definition. Or is the definition already gone somehow?

Reply:
Cool - thanks for taking the time to explain that.